html/browsers/history/the-history-interface/history_go_to_uri.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<title>history.go()</title>
+<script src="/resources/testharness.js"></script>
```

Comment:
Optionally add `<link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">` (but with your info instead). Preferably add `<link rel="help" href="https://html.spec.whatwg.org/multipage/browsers.html#dom-history-go">` and `<meta charset="utf-8">`.